src/main/java/com/software/modsen/passengermicroservice/controllers/PassengerRatingController.java

```diff
@@ -16,30 +16,24 @@
 import java.util.List;
 
 @RestController
-@RequestMapping(value = "/api/passenger/rating", produces = "application/json")
+@RequestMapping(value = "/api/passengers", produces = "application/json")
 @AllArgsConstructor
 @Tag(name = "Passenger rating controller", description = "Allows to interact with passenger ratings.")
 public class PassengerRatingController {
     private PassengerRatingService passengerRatingService;
     private final PassengerRatingMapper PASSENGER_RATING_MAPPER = PassengerRatingMapper.INSTANCE;
 
-    @GetMapping
+    @GetMapping("/ratings")
```

Comment:
Better to add "/all" part. Your route will be: /api/passengers/all/ratings

It's allowed to add such constructions in path: "me", "all", "list" etc, which replace the place of actual "id".